<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="hh" failures="1" tests="10" name="testNG1.ActitimeTestScenario1" time="46.311" errors="0" timestamp="2022-02-20T19:48:05 IST" skipped="2">
  <testcase classname="testNG1.ActitimeTestScenario1" name="test1" time="8.773"/>
  <system-out/>
  <testcase classname="testNG1.ActitimeTestScenario1" name="test2" time="6.267"/>
  <system-out/>
  <testcase classname="testNG1.ActitimeTestScenario1" name="test3" time="5.856"/>
  <system-out/>
  <testcase classname="testNG1.ActitimeTestScenario1" name="test3" time="4.631"/>
  <system-out/>
  <testcase classname="testNG1.ActitimeTestScenario1" name="test4" time="3.253"/>
  <system-out/>
  <testcase classname="testNG1.ActitimeTestScenario1" name="test4" time="3.165"/>
  <system-out/>
  <testcase classname="testNG1.ActitimeTestScenario1" name="test6" time="3.479">
    <failure message="null" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at testNG1.ActitimeTestScenario1.test6(ActitimeTestScenario1.java:154)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- test6 -->
  <system-out/>
  <testcase classname="testNG1.ActitimeTestScenario1" name="test7" time="10.886"/>
  <system-out/>
  <testcase classname="testNG1.ActitimeTestScenario1" name="test8" time="0.001">
    <skipped/>
  </testcase> <!-- test8 -->
  <system-out/>
  <testcase classname="testNG1.ActitimeTestScenario1" name="test5">
    <skipped/>
  </testcase> <!-- test5 -->
  <system-out/>
</testsuite> <!-- testNG1.ActitimeTestScenario1 -->
